requirements.txt

```diff
@@ -2,3 +2,9 @@ zipp>=0.5; python_version < "3.8"
 typing-extensions>=3.6.4; python_version < "3.8"
 pytest>=4.6.5
 mypy
+numpy
```

Comment:
It's really unfortunate that we need to require `numpy` just for that one instance check in `_update_from_parsed` - I don't suppose we can find an alternative for it? 

Replacing the occurrences of `numpy` in the test suite could be less of a problem, or we could put the tests behind flags and make sure `numpy` is installed on the CI but not required by the package in general.

Reply:
Since we check for `isinstance(value, numpy.ndarray)`, I don't think we can drop it. With putting the test behind flags you mean adding some pytest markers and only execute those tests when the marker has been specified?

Regarding making sure that dependencies are installed in the CI: I'd extend the `Build sdist` step in `azure-pipelines.yml` like this 
```yaml
  - script: |
      pip install -U -r requirements.txt
      pip install -U numpy wasabi
```
or 
```yaml
  - script: |
      pip install -U -r requirements.txt
      pip install -U -r requirements-test.txt
```
Or do we prefer another way to do this?

Reply:
For the instance check, one options that's slightly ugly but should work, is:
```
print("<class 'numpy.ndarray'>" in [str(t) for t in type(value).__mro__])
```

Reply:
> For the instance check, one options...
Ah, true. Nifty. Since `numpy` is only a test requirement anyway, do we want to keep it, only install it for the CI and document that correspondingly?

Reply:
Installing `numpy` and `pathy` in `azure-pipelines.yml` now and using `srsly.has_numpy` as flag for a `@pytest.mark.skipif` for the corresponding tests. Let me know if that's alright for you.